packages/realm/binding/apple/platform.mm

```diff
@@ -158,6 +163,18 @@
     }
 }
 
+void JsPlatformHelpers::after_realm_open(const std::shared_ptr<Realm> sharedRealm) {
+     if(sharedRealm->config().exclude_from_icloud_backup) {
+       RLMAddSkipBackupAttributeToItemAtPath(sharedRealm->config().path);
```

Comment:
Nit: It could make sense to introduce a local `const auto path = sharedRealm->config().path` to keep things DRY.